exporters/stdout/stdouttrace/trace_test.go

```diff
@@ -209,7 +222,7 @@ func TestExporterShutdownHonorsCancel(t *testing.T) {
 	innerCtx, innerCancel := context.WithCancel(ctx)
 	innerCancel()
 	err = e.Shutdown(innerCtx)
-	assert.ErrorIs(t, err, context.Canceled)
+	assert.NoError(t, err)
```

Comment:
Can you update the test name to `TestExporterShutdownIgnoresCancel`

Reply:
Done - renamed test method